ROADMAP.md

```diff
@@ -0,0 +1,13 @@
+# Roadmap
+
+Code formatting should be completely transparent for users. All the while, it should be customizable so users can define their own coding style if they want to.
+
+ocamlformat currently provides a large set of options and we will strive to support new options that the community cares about. However, we recognize that there's still a long road ahead to make the formatting experience seamless and well-integrated into the other development workflows.
+
+Based on these design principles for ocamlformat and the user feedback we've received, our roadmap to ocamlformat 1.0 includes:
+
+- **Versioning:** Ocamlformat 1.1 should be backward compatible and allow users to format code as they did with 1.0 if the version is specified in the configuration file.
+- **Customization:** Continue to support and add formatting options ocamlformat users care about.
+- **Concrete Syntax:** We'll migrate ocamlformat from an AST to a CST that contains location information to improve the formatting of comments.
```

Comment:
That's not the reason, the current AST contains locations and we are already capable of adding the missing locations to it.

The CST will make the code more robust to bugs by removing the need to guess the formatting of an abstract syntax that often have several possible syntaxes.
It could help with comments in an unspecified way, but that will not be related to locations.